packages/web3-providers/src/NFTScan/apis/TrendingAPI.ts

```diff
@@ -59,6 +74,40 @@ export class NFTScanTrendingAPI implements TrendingAPI.Provider<ChainId> {
         return response?.data ?? EMPTY_LIST
     }
 
+    async getCollectionOverview(
+        chainId: Web3Helper.ChainIdAll,
```

Comment:
You can follow other methods in this class.

```suggestion
        chainId: ChainId,
```